### PR TITLE
Sync changes for current installations

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Bug Fixes
- Fixes `synchronisation` for current installations

[Delivers [#120851421](https://www.pivotaltracker.com/story/show/120851421)]

### CODE REVIEW

**Comment by EvanBarbour3** [comment]
@wnowicki How's that..?

File: app/Basket/Synchronisation/InstallationSynchronisationService.php [hunk 1]
```diff
@@ -121,12 +122,17 @@ private function synchroniseNewInstallations(array $externalInstallations, Colle
...
 
                 try {
-                    $this->synchroniseInstallation($newInstallation->id);
+                    $this->synchroniseInstallation($newInstallation->id, true);
                 } catch (\Exception $e) {
                     // Empty
                 }
 
                 $rtn[] = 'New installation ' . $installation->getName() . ' has been added.';
+            } else {
```
Comment (wnowicki) [inline app/Basket/Synchronisation/InstallationSynchronisationService.php]: feels wrong that we are adding this in `synchroniseNewInstallations` method

Comment (EvanBarbour3) [inline app/Basket/Synchronisation/InstallationSynchronisationService.php]: Should be 'sychroniseInstallations', what do you think

**Comment by EvanBarbour3** [comment]
@wnowicki

**Pull request merged**